composer/conftest.py

```diff
@@ -19,6 +19,27 @@
 import pytest
 
 
+# this fixture initializes the Airflow DB once per session
+# it is used by DAGs in both the blogs and workflows directories, 
+# unless there exists a conftest at a lower level
+@pytest.fixture(scope="session")
+def airflow_database():
+    import airflow.utils.db
+
+    # We use separate directory for local db path per session
+    # by setting AIRFLOW_HOME env var, which is done in noxfile_config.py.
+
+    assert('AIRFLOW_HOME' in os.environ)
+
+    airflow_home = os.environ["AIRFLOW_HOME"]
+    airflow_db = f"{airflow_home}/airflow.db"
+
+    # reset both resets and initializes a new database
+    airflow.utils.db.resetdb()
```

Comment:
Good question - no - the blog post folder is staying as `rbac=None` because that's the Airflow 1 way and I am keeping that singular instance of code as Airflow 1. if we add `rbac=None` with Airflow 2 it will break. We could move this into the `workflows` folder, maybe? Not sure if it would mess up other tests though, haven't tried it. Honestly, I anticipate messing around with these configs a lot in the next few months.

Reply:
Resolving for now, but we can work together to make appropriate adjustments especially as we add more blog post code

Reply:
sounds good!